packages/nodes-base/nodes/Zoho/ZohoCrm.node.ts

```diff
@@ -34,7 +34,7 @@ export class ZohoCrm implements INodeType {
 		subtitle: '={{$parameter["operation"] + ": " + $parameter["resource"]}}',
 		group: ['input'],
 		version: 1,
-		description: 'Consume Zoho CRM API.',
+		description: 'Consume Zoho CRM API',
```

Comment:
CRM as part of the name? Do other CRM nodes have CRM in their names?

Reply:
there's agile crm and it has crm in the name